wip-stringparsexml.md

```diff
@@ -0,0 +1,174 @@
+<pre>
+  WIP: WIP-00XX
+  Layer: Consensus (hard fork)
+  Title: Add StringParseXMLMap to RADON operators
+  Authors: Luis Rubio <luisrwitnet.foundation>
+  Discussions-To: `#dev-general` channel on Witnet Community's Discord server
+  Status: Draft
+  Type: Standards Track
+  Created: 2021-12-14
+  License: BSD-2-Clause
+</pre>
+
+
+## Abstract
+
+The **StringParseXMLMap** is proposed to be included in the operators catalog available in the RAD engine. Data requests
+will then support XML parsing as part of the **retrieval** scripts.
+
+
+## Motivation and rationale
+
+In order to increase the flexibility and potential use cases of the Witnet protocol, it is paramount to continue 
+expanding the catalog of available operators.
+
+This document proposes adding support for **XML parsing** allowing the protocol to extract information from XML APIs
+or XML-encapsulated strings to be retrieved in Witnet by witnesses
+
+## Specification
+
+The *StringParseXMLMap* operator converts a *RadonString* (where the XML is defined) in a *RadonMap* following the next
+rules:
+- Every element will be defined as a *RadonMap* where the **tag** information would be inserted with the *_label* key, 
```

Comment:
What do you think of actually using `_tag` instead of `_label`? It looks like that's the proper XML name. Sorry for suggesting `label` in the first place.

```suggestion
- Every element MUST be turned into a `Map` in which the tag name will be inserted as the `_tag` key, 
```